dietpi/dietpi-drive_manager

```diff
@@ -998,17 +994,35 @@
 
 				fi
 
-			elif [ "$G_WHIP_RETURNED_VALUE" = "Move User data" ]; then
+			elif [ "$G_WHIP_RETURNED_VALUE" = 'Reserved blocks' ]; then
+
+				G_WHIP_INPUTBOX 'Ext4 formatted drives allow the reservation of drive space for the root user to assure system functionality, if filled by other users or processes, and to avoid fragmentation of large files.\n\nHowever, on modern drives, the default of 5% reserved blocks is often by orders of magnitude larger than necessary. You may want to reduce the percentage to an absulte reserved space of about 500 MiB, which should be enough, to enable root user starting and maintaining the system. Additionally, on non rootfs drives reserved blocks are not necessary at all.\n\nPlease enter the desired percentage of reserved blocks, e.g. "0.05" for 0.05% or "10" for 10%.\nNote: Only values between "0" and "50" are allowed.'
+				if (( ! $? )); then
+
+					if grep -qE '^[0-9]*\.?[0-9]*$' <<< "$G_WHIP_RETURNED_VALUE" &&
+						(( bc -l <<< "$G_WHIP_RETURNED_VALUE <= 50" )); then
+
+						G_RUN_CMD tune2fs -m "$G_WHIP_RETURNED_VALUE" "${aDRIVE_MOUNT_SOURCE[$INDEX_DRIVE_BEING_EDITED]}"
+
+					else
+
+						G_WHIP_MSG 'Error:\n\nPlease enter an integer or float between "0" and "50", e.g. "0.5" or "10".'
```

Comment:
@Fourdee 
Do you know a good way to let user directly retry the input, if entry was found wrong? Hmm just have the idea to use `while [ "$G_WHIP_RETURNED_VALUE" = 'Reserved blocks' ]` loop und do break if float check was passed?

Reply:
@MichaIng 

> just have the idea to use while [ "$G_WHIP_RETURNED_VALUE" = 'Reserved blocks' ] loop und do break if float check was passed?

Yep spot on 👍

Reply:
@Fourdee 
Done, jep that's the way I like it 😃.